README.md

```diff
@@ -1,14 +1,40 @@
 Here is the steps about how to install bestchains BaaS platform
 
+# Prerequisites
```

Comment:
1. add a 1st level title `Installer`
2. use 2rd level  title for `Prerequisites`...